srcpkgs/greetd/files/greetd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec greetd
```

Comment:
Should not we use something like this to make sure that it is ran by `_greeter`?

```
export USER="_greeter"
export HOME="/home/$USER"

groups="$(id -Gn "$USER" | tr ' ' ':')"

exec chpst -u "$USER:$groups" greetd
```

P.S. I'm quite new to Void Linux, so I'm sorry if I am missing something obvious :)

Reply:
I think I know the answer. The greetd is started as root, but it will start the command from its config as `_greeter`.

Reply:
Right, the suggestion you made at first is only necessary if the program does not change users and drop privileges on its own, such as when creating user-level services.